File: Listas de Exercícios (Python)/Python/Lista 2 - Casa/2.py
print("Número Inteiro - Distância (Número-chave)\n")

N=int(input("Informe um número de 0 a 100: "))

N_C=50
D=N-N_C

if(D<0):
    D=D*-1

print(f"\nA distância entre o número chave e o número informado é: {D}\n")

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/C/Lista 2 - Casa/2.c
#include <stdio.h>

int main()
{
    float N, N_C, D;
    
    printf ("Número Inteiro - Distância (Número-chave)\n\n");
    
    printf("Informe um número de 0 a 100: "); scanf("%f", &N);
    
    N_C=50;
    D=N-N_C;
    
    if (D<0) {
        D=D*-1;
    }
    
    printf("\nA distância entre o número chave e o número informado é: %.2f\n", D);
}